sbi-spec/src/lib.rs

```diff
@@ -46,6 +46,7 @@ pub mod cppc;
 pub mod nacl;
 // §16
 pub mod sta;
+pub mod fwft;
```

Comment:
```suggestion
// §18
pub mod fwft;
```

If no comment is added, rustfmt will rearrange order of modules, it will not be consistent to the order of chapters.